source/define-const-property.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const is = require('@sindresorhus/is');
+
+module.exports = (where, properties, deep = false, excluded = []) => {
```

Comment:
This method needs some internal docs in form of a code comment above. What it does exactly.

For readability, I think `deep = false, excluded = []` should be converted into an options object. `deep` is a boolean trap: https://ariya.io/2011/08/hall-of-api-shame-boolean-trap

Reply:
Thanks for telling me about that, very interesting :3